core/main.tf

```
@@ -12,3 +12,9 @@ module "ssm" {
     secret_key_base = var.secret_key_base
   }
 }
+
+module "clouwatch" {
```

```suggestion
module "cloudwatch" {
```

I am even more inclined to name it in relation to ECS because that's what this instance of CloudWatch logs will be used for when you implement ECS. ie ecs_logs as a name 🤔

---

I believe the module name should be the service name to be consistent with others.
Corrected the module name in [72c2665](https://github.com/nvminhtue/tyson_devops_ic/pull/26/commits/72c2665759dfea943a523ac814e0a71d53f3e0c6)

---

My view is that it's a particular _instance_ of your Cloudwatch resource from `/modules/cloudwatch`, which is just a log group. If you had multiple Cloudwatch log groups in your application, ie then needing multiple `module`s, you would have to change this name and give it context to what it's used for. This loggroup will only be used for ECS I think? 

Just my opinion but this is just a thin wrapper over the resource with the same name and doesn't offer anything extra. It could technically be removed from `/modules/cloudwatch` and placed here in `core/main without modulizaring it lol.

But okay, we can keep it the same name if you want 😆